content/guides/indicators-timesketch/infrastructure-setup.md

```diff
@@ -8,25 +8,87 @@ cascade:
 
 ## Before we start
 
-- You’ll be running two set of docker compose "projects". One for Yeti, and one
-  for timesketch;
-- You’ll connect the Timesketch and Yeti containers to the same network;
+Yeti offers two installation methods: [Kubernetes](#k8s-installation) (K8s) or [Docker](#docker-installation) (through
+`docker-compose`). Choose your preferred method and follow the instructions in
+this guide.
+
+## K8s Installation
+
+If you are new to Kubernetes, consider reviewing the OSDFIR Infrastructure
+[getting started guide](https://github.com/google/osdfir-infrastructure/blob/main/docs/getting-started.md).
+
+### Prerequisites
+
+To get started, ensure you have [Helm](https://helm.sh/docs/intro/install/) and
+[Kubectl](https://kubernetes.io/docs/tasks/tools/) installed and are
+authenticated to your Kubernetes cluster.
+
+{{< callout type="info" >}}
+
+**Note**: If you don't have a remote k8s setup, you can still use this
+[Minikube](https://minikube.sigs.k8s.io/docs/start/) or
+[KIND](https://kind.sigs.k8s.io/docs/user/quick-start/) to install Yeti and
+Timesketch locally.
+
+{{< /callout >}}
+
+### Pull chart and install release
+
+Once complete, add the repo containing the Helm charts as follows:
+
+```console
+helm repo add osdfir-charts https://google.github.io/osdfir-infrastructure
+```
+
+If you had already added this repo earlier, run `helm repo update` to retrieve
+the latest versions of the packages. You can then run
+`helm search repo osdfir-charts` to see the available charts.
+
+To install the Yeti and Timesketch chart using a release name of `my-release`:
+
+```console
+helm install my-release osdfir-charts/osdfir-infrastructure \
+--set global.turbinia.enabled=false
+```
```

Comment:
Will this also install turbinia, but not enable it? Might be good to say what happens here and why we're adding a flag referencing another project.

Reply:
Yeah fair enough, I've added a note right below the install command talking about what happens with this flag

Reply:
Nice, just wrapped it into a callout. LGTM!